backend/spec/requests/api/v1/keywords_spec.rb

```diff
@@ -5,23 +5,36 @@
   let!(:user_1)         { FactoryBot.create(:user) }
   let!(:token_1)        { FactoryBot.create(:access_token, application:, resource_owner_id: user_1.id) }
   let!(:headers_1)      { { Authorization: "Bearer #{token_1.token}" } }
+  let!(:user_2)         { FactoryBot.create(:user) }
+  let!(:token_2)        { FactoryBot.create(:access_token, application:, resource_owner_id: user_2.id) }
+  let!(:headers_2)      { { Authorization: "Bearer #{token_2.token}" } }
 
   describe "GET /index" do
     context 'when keywords exist' do
-      before { FactoryBot.create_list(:keyword, 2 ,user_id: user_1.id) }
+      before do 
+        FactoryBot.create_list(:keyword, 2 ,user_id: user_1.id)
+        FactoryBot.create_list(:keyword, 5 ,user_id: user_2.id)
```

Comment:
To ensure test suites remain scalable, creating the minimum required test data is best only. Creating a single keyword would still make the tests meaningful and valid. There is no benefit to creating 2 or 5 keywords.